recipe/meta.yaml

```diff
@@ -48,7 +48,6 @@ test:
     - pytest
     - pytest-xdist
     - mock  # [py27]
-  run_test.sh
```

Comment:
Yea it also does not work ;-)

I just copied it from the documentation of the `meta.yaml` file which is a bit unclear in this section: http://conda.pydata.org/docs/building/meta-yaml.html#run-test-script